docs/contributing/minutes.md

```diff
@@ -4,6 +4,96 @@ title: Global Syllabus Team Minutes
+- The app can be found [here](https://unique-selkie-38a362.netlify.app/)
+- Will really help with difficult concepts like the difference between `map` and `forEach`
+- Great to extend this kind of visualisation app to other things like async code and HTTP requests
+- Great to think about how we can use tooling to really get the most out of our trainees
+
+- Next steps... 👣
+  - Barath will continue using with LDN9
+  - Try it with a running cohort
+  - Get feedback and survey the trainees
+  - Possibly create a smaller version and make other methods discoverable ❓
+
+#### CYF cloud track 🌧️
+
+_(Raised by Sally)_
+
+- This is a Postgrad module
+- Pedro originally ran an informal version of this course - people would go away and study something and then come back to discuss it
+- Tension between teaching something interesting vs teach something step by step that is useful to a company
+- Trainees can use free tier AWS accounts - but will be billed if you make mistakes
+- [Plan has been created here](https://docs.google.com/document/d/1qsvH-C1vXvevcKflYb2o8Ny2mmY2W2foP_yNssvIkZE/edit) for the CYF cloud track
+- Possible dates: Saturday 17th / 24th September
+- Course is going to be more like a discussion with Q&A
+- Focus of module: have some sort of application that you need to deploy
+- Want to support unbanked students as part of digital inclusion so they can access platforms like AWS
+- Need to get the balance right between videos and resources and indepednent learning and too many passive videos
+- Sometimes trainees can get lost if the steps aren't clear and they're given too much independence - then have to introduce more structure
+- This ties in with some of the emerging issues re peer-2-peer learning
+
+#### Sprint planning / ticket taking 🎫
+
+_(Raised by Sally)_
+
+Autumn winter goals ⚾
+
+**Greg:** Syllabus work for JS1 / JS2 with Richard, Transition materials for Blockly
+**Ali:** Teacher training pair up, Develop teaching community in London,
+**Mitch:** MVP Teacher training write up,
+**Everyone**: Continuing to incorporate TDD into syllabus, Organise a meetup to discuss JS1/2/3 and our goals
+
+#### TDD 🧪
+
+_(Raised by Ali)_
+
+- Confusion around Enzyme raised by someone in Slack
+- TDD asked for more and more by trainees
+- Useful way to teach new stuff
+- Don't really teach how to write tests at the moment
+- Don't actually run the tests either too
+- Having tests that run in the browser
+- Tasks broken down with writing a test for each requirement:
+  - 1 a) Write a test that asserts some func returns `[]`
+  - 1 b) Write a test that asserts some other requirement ...
+- Scaffolding for how to write tests properly with red-green-refactor etc
+- Need to keep modelling good TDD practices over multiple weeks
+
+### ❗Actions
+
+- [] Schedule a separate session for Fundamentals
+- [] Form a module team for Fundamentals
+- [] Organise a discussion around TDD incorporation
+
```

Comment:
We did discuss organising a meetup focused on JS module restructuring in Jan, but maybe we should hold off on making an action until closer to the time?